### ISSUE TITLE
How to get the mask to input the loss function

### ISSUE THREAD

**Issue by dldl3D** [intel-isl/MiDaS#10]
Hi, 

I want to ask how I can get the mask to input the loss function? Cloud you release the training code and data generation script?

Thank you!

**Comment by ranftlr** [comment]
Mask generation is described in Section A of the supplementary material. Can you specify what exactly is unclear?

**Comment by dldl3D** [comment]
I find that the input of compute_scale_and_shift function is prediction, target, mask. How can I get the mask?
def compute_scale_and_shift(prediction, target, mask):
    # system matrix: A = [[a_00, a_01], [a_10, a_11]]
    a_00 = torch.sum(mask * prediction * prediction, (1, 2))
    a_01 = torch.sum(mask * prediction, (1, 2))
    a_11 = torch.sum(mask, (1, 2))

**Comment by ranftlr** [comment]
As mentioned before, we describe mask generation for our own dataset in Section A of the supplementary material:  "We perform a left-right consistency check and mark pixels with a disparity difference of more than 2 pixels as invalid". The same strategy is used for WSVD. The other datasets come with their own masks, please refer to their respective websites/papers for instructions.

**Comment by ranftlr** [comment]
Closing due to inactivity.
